### DISCUSSION TITLE
Advantages/disadvantages of stationary = TRUE

### DISCUSSION THREAD

**Discussion by storriel** [bmcclintock/momentuHMM#135] [Q&A] [answered]
I am using fitHMM with two data streams (step length and turning angle), and three states (ARS, intermediate, transit). I have separated the tracks into many segments (~100 per individual), as I had gaps in the data and wanted to retain only the highest accuracy Argos locations (plus GPS locations) prior to re-estimating locations using a correlated random walk and then fitting the HMM.  

Given that the tracks have been split at various times which are unrelated to behaviour, I would assume that the probability of the initial state being each of the three states would be equal to their probability over infinite time (stationary distribution). 

When I run fitHMM with stationary = FALSE, the initial distribution is ~ (0.92, 0.00003, 0.08). 
When looking through the decoded tracks and checking the step length and turning angle, there are some rare instances of a track beginning with the ARS state when perhaps it would be better represented by intermediate or transit. 

When I run fitHMM with stationary = TRUE, the initial distribution is (0.40, 0.31, 0.29), and examination of the decoded tracks reveals that some of them have been (more appropriately?) decoded as starting off in an intermediate state or transit state.  

The transition probabilities are similar for both models (<= 0.02 difference for each transition probability). And the step and angle parameter estimates are also similar. Most of the decoded tracks are identical, and neither run produces tracks with states that appear extremely improbable. 

But I wanted to check whether I have misunderstood this, and whether in such a scenario there are advantages/disadvantages to using stationary = TRUE vs FALSE? 

Many thanks for the help. And thanks for producing this brilliant package

**Comment by bmcclintock** [comment, marked as answer]
In the general case without constraints or covariates on the state transition and/or initial distribution parameters, it usually doesn't make a big difference in terms of inference. Based on your description, this seems to be the case here. Unless you're really unhappy about those handful of segments being initially assigned to what appears to you as the "wrong" state (note you might also want to look at the estimated state probabilities using `stateProbs` to get a sense for the uncertainty), I'd probably just suggest using AIC or BIC to select the best-supported model.

**Reply by storriel** [reply]
Good stuff, many thanks for the reply Brett
